doc/stdlib/io-network.rst

```diff
@@ -415,9 +415,7 @@ General I/O
    * use ``(key => value) in dict`` to see if this particular combination is in the properties set
    * use ``get(dict, key, default)`` to retrieve the most recent value for a particular key
 
-   .. code-block:: julia
-
-       IOContext(io::IO, context::IOContext)
+     IOContext(io::IO, context::IOContext)
```

Comment:
either would work, separating the docstrings could technically be a separate PR to be pedantic